### DISCUSSION TITLE
Prisma to connect to multiple datasources ? or DBs?

### DISCUSSION THREAD

**Discussion by Rahmo** [prisma/prisma#6001] [Q&A] [answered]
im not sure if its doable but can I connect to multiple data sources multiple DBs it would be so cool if its 
if so maybe I can get an example to that? 

Thanks you

**Comment by ryands17** [comment, marked as answer]
@Rahmo 👋 

You can use the workaround mentioned in this issue: https://github.com/prisma/prisma/issues/2443#issuecomment-630679118 for how to connect to multiple DB's.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
